Ulotka-informacyjna.md

```diff
@@ -0,0 +1,51 @@
+# HABATORY ROK
+
+## GŁÓWNA ULOTKA INFORMACYJNA
+
+### dla uczestników
+
+#### TEN DOKUMENT BĘDZIE MODYFIKOWANY
+
+## 📝 ARKUSZ WYJAZDOWY
+
+[arkusz wyjazdowy][aw] - tu proszę uzupełnić co potrzeba, znajdziecie w nim Harmonogram
+
+## ⭐ CELE SZKOLENIA
+
+- Świadomość swojego ciała w kajaku i poza nim
+- Wpływ pozycji ciała w kajaku na manewry
+- Metodyka wiosłowania
+  Znajomości sprzętu kajakowego oraz świadomego jego użycia
+- Doskonalenie techniki wchodzenia i wychodzenia z cofek
+  \*Zachowanie w odwoju
+- Umiejętności czytania wody
+- Wykorzystanie potencjału 66,6% polskich sztucznych torów slalomowych do rozwoju umiejętności kajakarskich
+- Integracja uczestników
+
+## 🗺️ LOKALIZACJA I TERMINY
+
+➡️ **Warszawa**
+
```

Comment:
Maybe this would look better in a table?

Reply:
ok no

Reply:
Why?